pkg/api/delete.go

```diff
@@ -2,13 +2,12 @@ package api
 
 import (
 	"fmt"
-	"net/http"
-
 	"github.com/NYTimes/gziphandler"
 	"github.com/prometheus/prometheus/promql/parser"
 	"github.com/timescale/promscale/pkg/log"
 	"github.com/timescale/promscale/pkg/pgclient"
 	"github.com/timescale/promscale/pkg/pgmodel"
+	"net/http"
```

Comment:
Minor nit: sort the imports by grouping standard and 3rd party imports (like it was) :relaxed: